tools/PI/DevHome.PI/Helpers/InternalToolsHelper.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+
+namespace DevHome.PI.Helpers;
+
+internal sealed class InternalToolsHelper
+{
+    public static readonly InternalToolsHelper Instance = new();
+
+    private readonly List<Tool> _allInternalTools;
+
+    public ReadOnlyCollection<Tool> AllInternalTools => _allInternalTools.AsReadOnly();
+
+    private InternalToolsHelper()
+    {
+        // All internal tools should be in this list
+        _allInternalTools = new List<Tool>
```

Comment:
Does the list need to be in a particular order?

Reply:
Alphabetical? Might be tricky with localization involved. Once I add tool #2, I'll need to figure that out.